unit_tests/rms_tests.cc

```diff
@@ -49,6 +49,13 @@ TEST(RMSTests, buffer) {
 
     // buffer should now be 0, 0, 0, 0, giving rms of 0
     ASSERT_EQ(rms.calc(), 0);
+
+    Audio sample{INT16_MAX, INT16_MAX, INT16_MAX, INT16_MAX};
+
+    rms.insert(sample);
+
+    // buffer should now be 0, 0, 0, 0, giving rms of 0
```

Comment:
comment is wrong

Reply:
so is this one, but it doesn't keep me up at night